include/audio_defines.h

```diff
@@ -51,6 +52,9 @@ struct audio_metadata {
 	/* Number of valid bits for a sample (bit depth). Typically 16 or 24. */
 	uint8_t bits_per_sample;
 
+	/* Number of bytes per channel (left/right etc) */
```

Comment:
Is this the number of bytes in the bitstream per active location?
If so can you update the comment to reflect this?
If it is not then all good and I'll need to fill this when decoding/encoding.